numba/typing/arraydecl.py

```diff
@@ -612,22 +612,38 @@ def sum_expand(self, args, kws):
     sum can be called with or without an axis parameter.
```

Comment:
This documentation line should probably be updated to reflect the new signature.

Reply:
fixed